docs/platforms/python/integrations/launchdarkly/index.mdx

```diff
@@ -44,3 +46,5 @@ sentry_sdk.capture_exception(Exception("Something went wrong!"))
 ```
 
 Visit the Sentry website and confirm that your error event has recorded the feature flag "hello" and its value "false".
+
+<PlatformContent includePath="feature-flags/change-tracking-cta-launchdarkly" />
```

Comment:
This is a lot of duplicate content from `/organization/integrations/feature-flag/launchdarkly/`

instead i think it would be clearer if this include has a smaller blurb and then linked over to ^ directly.